src/EventTickets/Migrations/CreateEventTicketsTable.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Give\EventTickets\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class CreateEventTicketsTable extends Migration {
+    /**
+     * @inheritdoc
+     */
+    public static function id() {
+        return 'give-events-create-events-tickets-table';
+    }
+
+    public static function title() {
+        return 'Create give_events table';
```

Comment:
Maybe a small typo here. I believe you meant to say "_Create give_event_tickets table_" here.

Reply:
Good catch - should be "Create give_event_tickets table".